cmd/boots/main.go

```diff
@@ -86,6 +86,9 @@ type config struct {
 	kubeAPI string
 	// kubeNamespace is an override for the namespace the kubernetes client will watch.
 	kubeNamespace string
+	// osiePathOverride allows a completely custom path/URL to be specified for OSIE/Hook images
+	// This will bypass the hardcoded path appending of 'misc/osie/current' to the path
+	osiePathOverride string
```

Comment:
this seems to not be used anywhere. I think it needs to be plumbed through to the `installer/osie/main.go`.